rimage/transform/distorter.go

```diff
@@ -17,10 +17,16 @@ const (
 // Distorter defines a Transform that takes an undistorted image and distorts it according to the model.
 type Distorter interface {
 	ModelType() DistortionType
+	CheckValid() error
 	Parameters() []float64
 	Transform(x, y float64) (float64, float64)
 }
 
+// ErrInvalidDistortionError is used when the distortion_parameters are invalid.
+func ErrInvalidDistortionError(msg string) error {
```

Comment:
[nit] I'd chose either `ErrInvalidDistortion` or `InvalidDistortionError` as both is redundant

Reply:
done